src/vdc_collection.rs

```diff
@@ -0,0 +1,205 @@
+
+/// Supported credential formats.
+#[derive(PartialEq)]
+pub enum CredentialFormat {
+    MDoc,
+    W3cJwtVc,
+    W3cJwtVcJsonld,
```

Comment:
Align with the format identifiers defined in the OID4VC specs.

```suggestion
    MsoMdoc,
    JwtVcJson,
    LdpVc,
```

Reply:
Fixed.